cmd/cert-rotation/main.go

```diff
@@ -17,20 +17,12 @@ package main
 import (
 	"context"
 	"fmt"
-	"net/http"
```

Comment:
For these commands, their business logic has been moved into pkg/cli almost copy-paste. I'll call out specific things inline where I made changes.

The new binaries are just stubs that "exec" the CLI. We could opt to delete these entirely instead.